apps/flame-defi/app/pool/modules/new-position/components/area-chart-with-range.tsx

```diff
@@ -1,109 +1,101 @@
 import React from "react";
-import ReactECharts from "echarts-for-react";
+import {
+  Area,
+  AreaChart,
+  ResponsiveContainer,
+  XAxis,
+  YAxis,
+  ReferenceLine,
+} from "recharts";
 
-export const AreaChartWithRange: React.FC = () => {
-  const option = {
-    backgroundColor: "#071520", // HSL(203, 45%, 4%) converted to hex
-    tooltip: {
-      trigger: "axis",
-    },
-    // We adjust the grid so our chart has some margins and won't clip the axis.
-    // The chart itself sits "behind" the dataZoom overlay.
-    grid: {
-      left: 50,
-      right: 30,
-      top: 50,
-      bottom: 50,
-    },
-    xAxis: {
-      type: "category",
-      boundaryGap: false,
-      data: [
-        120, 132, 101, 134, 90, 230, 210, 180, 222, 150, 210, 300, 250, 400,
-      ],
-    },
-    yAxis: {
-      type: "value",
-      splitLine: { show: false },
-      axisLabel: { show: false },
-    },
-    series: [
-      {
-        name: "Example Data",
-        type: "line",
-        smooth: false,
-        showSymbol: false,
-        areaStyle: {
-          color: "rgba(230, 149, 41, 0.2)",
-        },
-        lineStyle: {
-          color: "rgba(230, 149, 41, 1)",
-        },
-        itemStyle: {
-          color: "rgba(230, 149, 41, 1)",
-        },
-        data: [
-          120, 132, 101, 134, 90, 230, 210, 180, 222, 150, 210, 300, 250, 400,
-        ],
-      },
-    ],
-    // This dataZoom configuration creates a draggable range‐selector
-    // that covers the entire plotting area. We make it semi‐transparent
-    // so that the chart is still visible behind it.
-    dataZoom: [
-      {
-        type: "slider",
-        show: true,
-        // Fill the full chart area:
-        top: 0,
-        bottom: 0,
-        left: 0,
-        right: 0,
-        height: "100%",
+interface AreaChartWithRangeProps {
+  priceRange?: number[];
```

Comment:
This doesn't appear to ever be undefined. Is that just because you're always providing dev data while developing?